frontend/as_lp/components/visual-policy-builder.tsx

```
@@ -0,0 +1,593 @@
+  // Simple condition evaluator - in real implementation, this would be more sophisticated
+  try {
+    // Replace variable references with actual values
+    let expr = condition
+      .replace(/severity/g, context.severity?.toString() || '0')
+      .replace(/recurrence_count/g, context.recurrence_count?.toString() || '0')
+      .replace(/component_health/g, `'${context.component_health || 'healthy'}'`)
+      .replace(/resource_usage/g, context.resource_usage?.toString() || '0')
+
+    // Simple evaluation (in production, use a proper expression evaluator)
+    return eval(expr)
+  } catch {
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)